README.MD

```diff
@@ -0,0 +1,29 @@
+# Kafka Proto Parquet Writer
+Kafka proto parquet writer reads logs from a Kafka topic and writes them as parquet files. Parquet files can be created on local file system or HDFS. It can read and write logs in multiple threads. As writing to single parquet file can not be done concurrently, each threads writes to a separate file. 
```

Comment:
logs -> records

This is github! I should indicate the word which I want to change!